src/containers/Ledgers/test/LedgersPage.test.js

```diff
@@ -10,10 +10,33 @@ import { I18nextProvider } from 'react-i18next';
 import i18n from '../../../i18nTestConfig';
 import Ledgers from '../index';
 import ledgerMessage from './mock/ledger.json';
-import ledgerSummaryMessage from './mock/ledgerSummary.json';
 import validationMessage from './mock/validation.json';
-import metricMessage from './mock/metric.json';
 import { initialState } from '../../../rootReducer';
+import prevLedgerMessage from './mock/prevLedger.json';
+import moxiosData from './mock/rippled.json';
+
+class MockResponse {
+  constructor() {
+    this.iterations = 0;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get response() {
+    const request = moxios.requests.mostRecent();
+    const postParams = JSON.parse(request.config.data);
+    const { method } = postParams.options;
+    return moxiosData[method];
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  get status() {
+    return 200;
+  }
+}
```

Comment:
general mockresponse that maybe takes a path to a directory that contains all the JSON fixtures